test/e2e/functional/k8s-patch-basic.yaml

```diff
@@ -0,0 +1 @@
+examples/k8s-patch-basic.yaml
```

Comment:
These new YAML files in test/e2e/functional seem unnecessary since the ones in examples folder are already tested?

Reply:
I couldn't tell if they were or not.  I originally didn't symlink them but ran them with code that should have broken them (it broke locally) but it passed the tests in the PR.  Also I tried to search the output to see if "patch" was in there and I couldn't find it.  And I saw a lot of other files symlinked in a similar way.  So I tried it.  But I can remove those symlinks if you want or if you're sure they already get run. This is my first PR touching this part, so I'm not sure.

Reply:
@alexec Could you check if this is needed as you added the verifier.py annotation initially?

Reply:
Just took another look. I think this is fine. The path is incorrect though so might want to make sure it's executed.